e2e/wdio/platform/pages/combobox.po.ts

```diff
@@ -0,0 +1,162 @@
+import { BaseComponentPo } from './base-component.po';
+import { webDriver } from '../../driver/wdio';
+
+export class ComboBoxPo extends BaseComponentPo {
+    private url = '/combobox';
+    root = '#page-content';
+
+    comboBoxRoot = '.fdp-combobox';
+    comboBoxDropdownExpanded = '.fd-list';
+    groupHeader = '.fd-list__group-header';
+    comboboxWithGroup = 'input[ng-reflect-name="group"]';
+    comboboxTwoColumns = '[name="columns"] input';
+    optionsArray = '.fd-list li'
+    comboBoxInput = this.comboBoxRoot + ' input';
+
+    selectedDropDownOption = (name: string) => {
+        return `//span[contains(.,'${name}')]//ancestor::li[@role="listitem" and contains(@class, "is-selected")]`
+    }
+
+    dropDownOption = (name: string) => {
+        return `//span[contains(.,'${name}')]//ancestor::li[@role="listitem"]`
+    }
+
+    comboBoxOptionHint = (typedCharacters: string, restCharacters: string) => {
+        return `//span[text()='${restCharacters}']//strong[text() = '${typedCharacters}']`
+    }
+
+    comboBoxButtons = (name: string) => {
+        return `//label[@for='${name}']/following-sibling::fdp-input-message-group//button`
+    }
+
+    comboBoxExpandedButtons = (name: string) => {
+        return `//label[@for='${name}']/following-sibling::fdp-input-message-group//button[contains (@class,"is-expanded")]`
+    }
+
+    comboBoxInputs = (name: string) => {
+        return `//label[@for='${name}']/following-sibling::fdp-input-message-group//input`
+    }
+
+    filledComboBoxInputs = (name: string , option: string) => {
+        return `//label[@for='${name}']/following-sibling::fdp-input-message-group//input[@ng-reflect-model='${option}']`
+    }
+
+    getDropdownOptionsText(type: string): string[] {
+        let textArr: string[] = [];
+            this.expandDropdown(type);
+            const optionsLength = webDriver.getElementArrayLength(this.optionsArray);
```

Comment:
`return webDriver.getTextArr(this.optionsArray, 0,-1)`
Perhaps need to pull last changes from feature/e2e-test-migrate-to-wdio